purchase.adoc

```diff
@@ -388,288 +517,533 @@ Fields named "*UUID1" are the standard UUID Type 1 (8-4-4-4-12) representation o
+            "percentage": 50,
+            "quantity": 1
+        }
+    ]
 
-Payment types are added when product offerings at iZettle changes, so for forwards-compatibility it is important that clients are tolerant of receiving payments of types that are not defined upfront.
 
-To be able to represent all types of payments it has an `attributes` map that contain all payment type specific data such as `maskedPan` or `cardType` for card payments or `handedAmount` for cash payments.
-Other payments such as Swish, Vipps and MobilePay don't have attributes. They will only be represented by a `uuid`, an `amount` and a `type`.
+----
+`name` attribute is optional, not always present.
+For more information see section "Discounts".
+|serviceCharge|object a|Service charge added to a purchase e.g. delivery charge. +
+Example:
+[source,json]
+----
+ "serviceCharge": {
+    "amount": 499,
+    "title": "Standard Shipping",
+    "vatPercentage": 16.0,
+    "quantity": "1"
+  }
+----
 
-._example of cash payment_
-```json
-{
-  "uuid": "ce90dc90-dcaa-11e6-87a4-0cd119752226",
-  "amount": 1000,
-  "type": "IZETTLE_CASH",
-  "attributes": {
-    "handedAmount": 1000
+|created|string|Created timestamp according to the https://en.wikipedia.org/wiki/ISO_8601[ISO 8601] format e.g. "2019-12-03T17:05:06.123+0000".
+|refund|boolean|Indicates if given purchase is a refund.
+|refunded|boolean|Indicates if given purchase has been refunded.
+|refundsPurchaseUUID|string|Unique id of the purchase that was refunded as string.
+|refundsPurchaseUUID1|string|Unique id of the purchase that was refunded as UUID version 1.
+|refundedByPurchaseUUIDs|array a|List with ids of purchases that refunded this particular one. +
+[source, json]
+----
+"refundedByPurchaseUUIDs": [
+    "PSITZxySSY2elO9cxPRdYQ",
+    "431jwpevS-y2rBzyEvl9kA"
+]
+----
+|refundedByPurchaseUUIDs1|array a|List with ids of purchases that refunded this particular one as UUIDs version 1. +
+[source, json]
+----
+"refundedByPurchaseUUIDs1": [
+    "3d221367-1c92-498d-9e94-ef5cc4f45d61",
+    "e37d63c2-97af-4bec-b6ac-1cf212f97d90"
+]
+----
+|groupedVatAmounts|object a|Contains VAT amounts grouped by percentage. +
+Example:
+[source, json]
+----
+"groupedVatAmounts": {
+    "25.0": 70000,
+    "12.0": 5000
+  }
+----
+|userDisplayName|string|Name of the user that took the payment.
+|gpsCoordinates|object a|Contains GPS coordinates of the location where purchase was made. +
+Example:
+[source,json]
+----
+"gpsCoordinates": {
+    "longitude": 19.80452501310729,
+    "latitude": 66.609375,
+    "accuracyMeters": 165
   }
+----
+|userId|integer|Unique id for user within iZettle system. Internal use.
+|cashRegister|object a|Cash register reference. Cash register is a service provided by iZettle and used by merchant in order to comply with requirements coming from the tax authority(only available in Sweden, Norway, Germany and France). +
+For now no public API is available for cash register. Mostly for internal use.
```

Comment:
Maybe we don't need to mention cash register API is mostly for internal use, since there's no public API for it?

Reply:
We get many questions regarding Cash register being available as public APIs. Thought it could be good to answer this question here right away instead of only placing it in FAQ. But maybe we can just say that this attribute is "Internal use only". What do you think @Astro-Choco ?

Reply:
Aha, sorry Natalia, I didn't know the context. Then it's good to be explicit as you suggested. For example, cash register is for internal use only. There is no public API available for cash register.